noir-projects/aztec-nr/aztec/src/encrypted_logs/payload.nr

```diff
@@ -122,23 +135,12 @@ fn compute_encrypted_log<let P: u32, let M: u32>(
     let outgoing_header_ciphertext: [u8; 48] = header.compute_ciphertext(eph_sk, ovpk);
     let incoming_body_ciphertext =
         compute_incoming_body_ciphertext(plaintext, eph_sk, recipient.to_address_point());
-    let outgoing_body_ciphertext: [u8; 144] =
+    let outgoing_body_ciphertext: [u8; 112] =
         compute_outgoing_body_ciphertext(recipient, fr_to_fq(ovsk_app), eph_sk, eph_pk);
 
     let mut encrypted_bytes = [0; M];
     let mut offset = 0;
 
-    // We assume that the sender wants for the recipient to find the tagged note, and therefore that they will cooperate
-    // and use the correct tag. Usage of a bad tag will result in the recipient not being able to find the note
-    // automatically.
-    let tag_bytes = unsafe { get_app_tag_bytes_as_sender(sender, recipient) };
-    increment_app_tagging_secret_index_as_sender(sender, recipient);
-
-    for i in 0..32 {
-        encrypted_bytes[offset + i] = tag_bytes[i];
```

Comment:
Tag is no longer part of the "encrypted" bytes.
Instead, it's the first field of the private log: `[tag, ...bytes_to_fields(encrypted_bytes)]`, concatenated in the above function `compute_private_log_payload`.
When it was part of the encrypted_bytes, bytes_to_fields would split it and leave the first 31 bytes in the first field, and the last byte in the second field, which made it really ugly when requesting data using siloed tags.